### PR TITLE
[CP-1797] Low battery connection with mc is possible

### PR DESCRIPTION
**Pull request by lukasmas**
Add a reason field to USBSecurityEndpoint reson when device is locked.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [x] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

**Review by patryk-sierzega — APPROVED** [review]

**Review by dkarski — APPROVED** [review]
LGTM ✅ (from the documentation perspective)

File: doc/os_api/endpoints/pure/security_endpoint.md [hunk 1]
```diff
@@ -32,16 +32,44 @@ Security endpoint (13)
...
+- *status* - 423 when phone is unlocked or 403 when phone is locked
+- *reason* - block reason, see code snippets below
+
+```c++
+    enum class BlockReason
+    {
+        EulaNotAccepted = 2,
+        BatteryCriticalLevel
+    };
+```
```
Comment (SP2FET) [inline doc/os_api/endpoints/pure/security_endpoint.md:69]: shouldn't we mention that those status codes are HTTP codes? If it's already written down then just resolve the comment :)

**Review by SP2FET — APPROVED** [review]

File: module-services/service-desktop/endpoints/security/SecurityEndpointHelper.cpp [hunk 2]
```diff
@@ -48,31 +48,31 @@ namespace sdesktop::endpoints
         return {sent::no, ResponseContext{.status = http::Code::BadRequest}};
```
Comment (SP2FET) [inline module-services/service-desktop/endpoints/security/SecurityEndpointHelper.cpp:48]: please change all returns to be in the same manner :)

**Review by Maciej-Mudita — APPROVED** [review]

File: doc/os_api/endpoints/pure/security_endpoint.md (hunk 1 above)
Comment (lukasmas) [reply doc/os_api/endpoints/pure/security_endpoint.md:69]: It is written here:
https://github.com/mudita/MuditaOS/blob/ff31f28e047a0e55878e091fe08c05854a42a6ac/doc/os_api/protocol_description/common_parameters_explanation.md

> status - HTTP-like status code

**Pull request merged**